documentation/common/configuring-using-restapi.adoc

```diff
@@ -96,11 +104,12 @@ object.
 }
 ----
 
-. You can then create those addresses using the REST API:
+. You can then create those addresses using the REST API (Replace `default` with the name of your address space if running in multitenant mode):
```

Comment:
Change to: "You can then create those addresses using the following REST API command. Replace `default` with the name of your address space if running in multitenant mode."